packages/genesys-spark-components/src/components/stable/gux-form-field/functional-components/gux-form-field-label/gux-form-field-label.tsx

```diff
@@ -1,25 +1,32 @@
 import { FunctionalComponent, h, VNode } from '@stencil/core';
 
-import { GuxFormFieldLabelPosition } from '../../gux-form-field.types';
+import {
+  GuxFormFieldLabelPosition,
+  GuxFormFieldIndicatorMark
+} from '../../gux-form-field.types';
 
 interface GuxFormFieldLabelProps {
   position: GuxFormFieldLabelPosition;
   required: boolean;
+  indicatorMark?: GuxFormFieldIndicatorMark;
 }
 
 export const GuxFormFieldLabel: FunctionalComponent<GuxFormFieldLabelProps> = (
-  { position, required },
+  { position, required, indicatorMark = 'required' },
```

Comment:
thoughts on setting the default value of `required` on line 11 to make it more clear?